src/components/requests/drawing.vue

```diff
@@ -1,6 +1,12 @@
+<style scoped>
+  code {
+    font-size: 100%;
+  }
+</style>
```

Comment:
In all the other example style is the last element. Move it there for consistency.